docs/en/serverless/projects/billing.asciidoc

```diff
@@ -11,11 +11,16 @@ Your monthly bill is based on the capabilities you use.
 When you use {obs-serverless}, your bill is calculated based on data volume, which has these components:
 
 * **Ingest** — Measured by the number of GB of log/event/info data that you send to your Observability project over the course of a month.
-* **Storage/Retention** — This is known as Search AI Lake.
-* In addition to the core ingest and retention dimensions, there is an optional charge to execute synthetic monitors on our testing infrastructure.
-Browser (journey) based tests are charged on a per-test-run basis,
-and Ping (lightweight) tests have an all-you-can-use model per location used.
+* **Retention** — Measured by the total amount of ingested data stored in your Observability project.
 
-For more information, refer to <<general-serverless-billing>>.
+Ingest and retention metering is based on the uncompressed, normalized, fully enriched data volume you ingest into your Serverless project. This allows you flexibility and consistency when choosing your preferred ingest architecture for enrichment, whether through {agent}, {ls}, OpenTelemetry, or collectors—without considering the possible impact on the cost.
```

Comment:
reading this now seems awkward to me...

> ...--without considering the possible impact on the cost.

maybe ...--without need to to consider possible impacts on the cost.

Reply:
Yah I think it's fine grammatically, but someone skim reading might take the wrong meaning.